### PR TITLE
More details on pending applications in user profile

### PR DESCRIPTION
**Pull request by hero101**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: src/domain/community/application/application.entity.ts [hunk 1]
```diff
@@ -19,6 +19,12 @@ export class Application extends AuthorizableEntity implements IApplication {
   @Column()
   ecoverseID?: string;
 
+  @Column()
+  createdDate!: Date;
+
+  @Column()
+  updatedDate!: Date;
+
```
Comment (atstoyanov) [inline src/domain/community/application/application.entity.ts]: Is there a way to put this in the base entity?

**Review by atstoyanov — CHANGES_REQUESTED** [review]

File: src/domain/community/application/application.entity.ts (hunk 1 above)
Comment (hero101) [reply src/domain/community/application/application.entity.ts]: @techsmyth, should I?

Comment (techsmyth) [reply src/domain/community/application/application.entity.ts]: It is already there in the base entity. 
Application extends AuthorizableEntity 
AuthorizableEntity extends BaseAlkemioEntity:

image

So the updates to the application.entity file are not required. The updates to application.interface are required in order to allow those fields to be visible on the api.

**Review by techsmyth — APPROVED** [review]
Nice!

**Pull request merged**